python/whylogs/core/metrics/metrics.py

```diff
@@ -273,7 +273,7 @@ def columnar_update(self, view: PreprocessedColumn) -> OperationResult:
                     if n_b > 1:
                         n_b = len(arr)
                         mean_b = arr.mean()
-                        m2_b = arr.var() * (n_b - 1)
+                        m2_b = arr.var(ddof=1) * (n_b - 1)
```

Comment:
It looks like v0 always did the sample variance (1 degree of freedom). So I take it we're declaring the v1 using population variance sometimes the bug and changing to sample variances in all cases despite the minor difference from previously computed profiles?

That works for me. I like the tests here, so I vote we merge this one and I'll rebase @bernease's PR on this one.

Reply:
correct, and also considering our bug of letting pandas series be stored in the NumpyView was a most common code path, the effective ddof was 1 when logging with pandas dataframes (which also covers pyspark).